src/Microsoft.Health.Dicom.Api/Logging/TelemetryInitializer.cs

```diff
@@ -0,0 +1,49 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Microsoft.Health.Dicom.Api.Logging;
+
+/// <summary>
+/// Class extends AppInsights telemtry to include custom properties
+/// </summary>
+public class TelemetryInitializer : ITelemetryInitializer
```

Comment:
Maybe a more specific name lol. Perhaps `[Api]VersionTelemetryInitializer`?

Reply:
Keep it generic so that the same class can be used to extend other properties